docs/UserGuide.md

```diff
@@ -80,14 +80,16 @@ SCRUMptious is a command-line project management software, designed for project
     WELCOME TO SCRUMPTIOUS
     command>
     ```
-1. Type the command `project /create -title Scrumptious -desc A java project -end 30 -sd 10` 
+1. Type the command `project /create -title Scrumptious -desc A java project -dur 30 -sd 10` 
 into the command line and press `Enter` to execute it.
-1. If the setup is correct, you can view the project by executing `project /view`:
+1. If the setup is correct, you will be able to view the project by executing `project /view`:
    ```
-   command> project /create -title Scrumptious -desc A java project -end 30 -sd 10
+   command> project /create -title Scrumptious -desc A java project -dur 30 -sd 10
    Project successfully created.
+       Title: Scrumptious
```

Comment:
This line is wrong, this is what I got when I run it
```
Project successfully created.
============================ PROJECT ============================
[ID: 1]
[Title: Scrumptious]
[Description: A java project]
[No members added]
[Project will start along with the first sprint]
[Project backlog is empty]
[There are no Sprints]
=================================================================
```

Reply:
No I changed the code, so did you try running with the updated code?

Reply:
Ran it with v2.0 jar file that's why I didn't see the change. Sorry about that. Looks good to me and can be merged.